open_instruct/dpo_tune.py

```diff
@@ -523,7 +524,7 @@ def load_model():
     args.num_train_epochs = math.ceil(args.max_train_steps / num_update_steps_per_epoch)
 
     # Figure out how many steps we should save the Accelerator states
-    checkpointing_steps = args.checkpointing_steps
+    checkpointing_steps = str(args.checkpointing_steps)
```

Comment:
I have to convert this to `str` because if the yaml file gives a 1000, `type(args.checkpointing_steps)` was actually int, so `checkpointing_steps.lower()` would error out.